packages/salesforcedx-vscode-lwc/test/vscode-integration/hover.test.ts

```diff
@@ -25,11 +27,33 @@ describe('LWC Hovers', () => {
     'lwc'
   );
 
+  let client: LanguageClient;
+
+  before(() => {
+    client = createLanguageClient(
+      path.join(
+        __dirname,
+        '..',
+        '..',
+        '..',
+        'node_modules',
+        '@salesforce',
+        'lwc-language-server',
+        'lib',
+        'server.js'
+      )
+    );
+    client.start();
+  });
+
   afterEach(async () => {
     await commands.executeCommand('workbench.action.closeActiveEditor');
   });
 
+  after(() => client.stop());
+
   it('Should provide additional details when hovering over a LWC tag', async () => {
+    await client.onReady();
```

Comment:
Are there other tests we should update (later) with these changes ?

Reply:
Possibly the intellisense one, but not sure if its worth changing now or if it becomes a problem. What do you think?

Reply:
I agree in not changing them right now, just asking so we can log a work item and get to them in the next couple of weeks.

Reply:
I'll log something to track it